daemon/cmd/agenthealth.go

```diff
@@ -42,6 +43,7 @@ func (d *Daemon) startAgentHealthHTTPService() {
 		statusCode := http.StatusOK
 		sr := d.getStatus(true)
 		if isUnhealthy(&sr) {
+			log.WithField(logfields.State, sr.Cilium.State).Warnf("/healthz returning unhealthy: %s", sr.Cilium.Msg)
```

Comment:
```suggestion
			log.WithFields(logrus.Fields{
			    logfields.State, sr.Cilium.State,
			    logfields.Message: sr.Cilium.Msg,
			).Warn("/healthz returning unhealthy")
```

Reply:
@christarazi done. Didn't find an existing "Message" logfield so added that also.

Reply:
I realized that `message` is also the field name that a log msg uses anyway, so something like this

```
level=warn message="This is the content of the log msg"
```

so I think we'd probably want to use the error field instead. Logrus has a helper called `WithError()` that can be called before the `WithFields`. Apologies for the confusion :)

Reply:
Please also squash all commits into the original.